provider/resource_keycloak_openid_client_authorization_role_policy.go

```diff
@@ -0,0 +1,193 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOpenidClientAuthorizationRolePolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationRolePolicyCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationRolePolicyRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationRolePolicyDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationRolePolicyUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakOpenidClientAuthorizationRolePolicyImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"resource_server_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"decision_strategy": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"logic": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "POSITIVE",
+			},
+			"scopes": {
+				Type:     schema.TypeSet,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Computed: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"role": {
```

Comment:
I can see in the GUI that this policy supports realm roles and client roles, which is this attribute supposed to represent?  And it looks like I can add more than one role to the policy, is there a reason you specified `MinItems` to be 1?

Reply:
I just checked the Chrome network inspector when creating a role policy, and I see that while bifurcated in the UI, the API call combines all the roles into one object:
![image](https://user-images.githubusercontent.com/50674203/75912308-fb844180-5e1e-11ea-8880-820077dffc3e.png)

Note the request above had one realm role and one client role.

Reply:
I think the reason for min items equaling 1 is because a role policy essentially does nothing if there are no roles specified.

Reply:
Thanks Ryan! 
@mrparkers I can remove the `MinItems` if you'd like.

Reply:
My bad, I misread `MinItems` to be `MaxItems`.  Thanks for the clarification on the roles.